composer/utils/checkpoint.py

```diff
@@ -125,6 +126,25 @@ def load_checkpoint(
             Ignored if the checkpoint is a local file path. (default: ``1_048_576`` bytes (1 MB))
         progress_bar (bool, optional): Whether or not to show a progress bar when downloading checkpoints.
             Ignored if the checkpoint is a local file path. (default: ``True``)
+        ignore_keys (List[str] | (Dict) -> None, optional): A list of paths for the ``state_dict`` of the checkpoint,
+            which, when provided, will be ignored from the state_dict before a checkpoint is loaded. Each path is a list
+            of strings specifying the keys to index into ``state_dict`` joined together with `/` as a seperator.
+            If a prefix is provided, all children are also ignored (as demonstrated in Example 2). See :mod:`composer.core.state`
+            for the structure of state_dict.
+
+            Example 1: `load_ignore_model_keys = ["state/model/layer1/weights", "state/model/layer1/bias"]` would ignore
```

Comment:
For readability, can these examples be converted into one big code block?
In addition, the variables here are `ignore_keys`, not `load_ignore_model_keys`.

Reply:
Sorry, what exactly do you mean by convert into one big code block?

Reply:
Nvm; I thought there would be more a a functional api we could call, but scratch this.